rosa_hcp/rosa-hcp-shared-vpc-config.adoc

```diff
@@ -0,0 +1,48 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="rosa-hcp-shared-vpc-config"]
+= Configuring a shared VPC for ROSA with HCP clusters
+:context: rosa-shared-vpc-config
+
+toc::[]
+
+You can create {hcp-title-first} clusters in shared,centrally-managed AWS virtual private clouds (VPCs).
+
+[NOTE]
+====
+This process requires *two separate* AWS accounts that belong to the same AWS organization. One account functions as the VPC-owning AWS account (*VPC Owner*), while the other account creates the cluster in the cluster-creating AWS account (*Cluster Creator*).
+====
+
+image::522-shared-vpc-overview.png[]
+
+* The hosted zones can be created in either the centrally-managed VPC account or in the workload account in which the cluster is deployed.
+
+.Prerequisites for the *VPC Owner*
+* You have an AWS account with the proper permissions to create roles and share resources.
+* The *Cluster Creator's* AWS account is separate from the *VPC Owner's* AWS account.
+* Both AWS accounts belong to the same AWS organization.
+* You enabled resource sharing from the management account for your organization.
+* You have access to the link:https://signin.aws.amazon.com[AWS console].
+
+.Prerequisites for the *Cluster Creator*
+* You installed the link:https://console.redhat.com/openshift/downloads#tool-rosa[ROSA CLI (`rosa`)] 1.2.26 or later.
+* You created all of the required link:https://docs.openshift.com/rosa/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.html[ROSA account roles] for creating a cluster.
+* The *Cluster Creator's* AWS account is separate from the *VPC Owner's* AWS account.
```

Comment:
(asked about the necessity of this earlier)

Reply:
I think we should remove this line.